database/exip.sql

```diff
@@ -624,6 +627,19 @@ VALUES
 /*!40000 ALTER TABLE `DeclarationHowDataWillBeUsed` ENABLE KEYS */;
 UNLOCK TABLES;
 
+# Dump of table DifferentTradingAddress
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `DifferentTradingAddress`;
+
+CREATE TABLE `DifferentTradingAddress` (
+  `id` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+  `application` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Comment:
Can we have this as a company relationship instead of application?

In the future there maybe functionality where if a user creates a new application and we have their company/business details, we can pre-populate things; Changing to have a company relationship instead of application would be cleaner.